### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by mjsimon91**
The front end for the profile has been made as a framework. We can work on future enhancements to the UI as well as use actual data to dynamically populate the pages. This is all hardcoded for now. The routes are also hardcoded for an id of "1" which will need to be changed to be ":id" in the future.

### CODE REVIEW

File: routes/api-routes.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  app.get("/api/members/:id", function(req, res){
+		db.Members.findOne({
+			where: {
+				id: req.params.id
+			},
+		}).then(function(dbMembers){
+			res.json(dbMembers);
+		});
+	});
+};
```
Comment (mbychkowski) [inline routes/api-routes.js]: Is `api-routes.js` being used anymore? I have been using `html-routes.js` and `listings/members-api-routes.js`

File: views/profileMessages.handlebars [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+              <p>3/1/18</p>
+            </div>
+          </div>
+        </div>
+      </div>
+     </div>
+
+    </div>
+ </div>
+</div>
```
Comment (mbychkowski) [inline views/profileMessages.handlebars]: I think these views are really good. Next steps to add routes and functionalitity.

File: models/members.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+  //Associate this with the Homes model
+  // Members.associate = function(models){
+  //   Members.hasMany(models.Home, {
+  //     onDelete: "cascade"
+  //   });
+  // };
+
+  return Members
+};
```
[the remark this answers is not in the document]
Comment (mjsimon91) [reply models/members.js]: Can you elaborate on this please? Does "include" not happen when you query data? This is posting data via sequelize

File: routes/api-routes.js (hunk 1 above)
Comment (mjsimon91) [reply routes/api-routes.js]: Going to remove this...was using this for some testing on my device

**Pull request closed without merging**